src/Service/Services/RestService.cs

```diff
@@ -84,6 +84,8 @@ RuntimeConfigProvider runtimeConfigProvider
                 requestBody = await reader.ReadToEndAsync();
             }
 
+            RequestValidator.ValidateEmptyRequestBodyForFindApi(operationType, requestBody);
```

Comment:
While the intention of this PR is good, we were ignoring GET request payloads to adhere to this RFC: https://www.rfc-editor.org/rfc/rfc7231#page-24

>  A payload within a GET request message has no defined semantics;
>    sending a payload body on a GET request might cause some existing
>    implementations to reject the request.

Some discussion around this in this stack overflow: https://stackoverflow.com/questions/978061/http-get-with-request-body

Reply:
Thanks for sharing the RFC and discussions!